### ISSUE TITLE
Trojan in esprima-python, that is generating OpenAPI ?

### ISSUE THREAD

**Issue by xet7** [wekan/wekan#2545]
@bentiss 

Your OpenAPI generation was added at https://github.com/wekan/wekan/pull/1965

My ESET NOD32 Antivirus for Linux complains that there is  JS/TrojanDownloader.Agent.NBT trojan at this file:
https://github.com/Kronuz/esprima-python/blob/master/test/fixtures/esprima-python/recursion.js

It could be related to this, that after running OpenAPI docs generation, there appeared my-directory-list.txt in many of my directories. To delete those, I had to type:
```
locate my-directory-list.txt | xargs rm
```
Can you check these?

**Comment by xet7** [comment]
I uploaded that file to virustotal. Looking at these links, it could be WannaCry virus that encrypts files etc. Anyway, the purpose of esprima-python and OpenAPI code is to just generate documentation to .html and .yml files, and I did not find any viruses from Wekan release files with ESET NOD32 and Windows Defender. Anyway, that recursion.js was detected as malicious by both ESET NOD32 and Windows Defender.

https://www.virustotal.com/gui/file/6944691f075c7a5788fa2a58bf56fe57717d9891597d8fdd586333a276bcdd1d/detection

https://answers.microsoft.com/en-us/protect/forum/all/win32exploitcve-2017-0147a-trojan/8b9e3090-0c56-4203-a2ab-859e9f8769a8

https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Ransom:Win32/WannaCrypt
